.github/workflows/ci.yaml

```diff
@@ -84,16 +84,33 @@ jobs:
       # See https://doc.rust-lang.org/nightly/cargo/guide/cargo-home.html#caching-the-cargo-home-in-ci
       - name: Cache Rust artifacts
         uses: actions/cache@v2
+        if: |
+          ${{ matrix.cmd == 'run-cargo-clippy' || matrix.cmd == 'run-functions-examples --application-variant=rust' }}
```

Comment:
Did caching work? Because if I understand the last CI run correctly, run-cargo-clippy took still 36 min (https://github.com/project-oak/oak/runs/4769340244?check_suite_focus=true). We'd expecting this to be less, no? But it seems we just kicked off a run again!

Reply:
I got a run with cargo clippy running in 4 mins instead of 40 mins (with no changes to the code), but now it's hard to reproduce because the cache keeps being evicted by other jobs. I think we'll have to submit this and see how it goes to have more useful data.

Reply:
Sounds good to submit and observe! 

But should we maybe note somewhere (e.g., the title), that the sccache is only working for run-cargo-clippy and Rust run-function-examples? We still may want to look into run-cargo-tests, if this works well, no?

Reply:
SG, I think the next thing to explore would be the single workspace though, there is just not enough space available to enable it for another step, otherwise I would do that already.